### PR TITLE
Feature: getting details of user on ../profile/me

### PR DESCRIPTION
**Pull request by vipulrawat**

### CODE REVIEW

File: packages/api/src/routes/profileRoutes.js [hunk 1]
```diff
@@ -1,8 +1,40 @@
...
+const { User, Batch } = require('../db');
 const ERR_MSGS = require('../../constants/ERR_MSGS');
 const profileValidation = require('../services/profileValidation');
 const { isAuthenticated } = require('../helper/auth/isAuthenticated');
+const { extractUser } = require('../helper/user');
+
+profileRoutes.get('/user/:id', isAuthenticated, extractUser, async (req, res) => {
+  const { id } = req.params;
+  const { user } = req;
+  if (user) {
```
Comment (bharaninb) [inline packages/api/src/routes/profileRoutes.js]: Try to check negativity first. It'll reduce lots of indentation.
Also, it increases code readability

Comment (vipulrawat) [reply packages/api/src/routes/profileRoutes.js]: Yup. changed it in next Commit.

File: packages/api/src/routes/profileRoutes.js [hunk 2]
```diff
@@ -1,8 +1,40 @@
...
+profileRoutes.get('/user/:id', isAuthenticated, extractUser, async (req, res) => {
+  const { id } = req.params;
+  const { user } = req;
+  if (!user) {
+    res.status(400).json({ error: ERR_MSGS.profileNotExist });
+  }
+  const {
+    email, firstName, lastName, profilePicUrl, batchId, role,
+  } = user;
+  if (id.toLowerCase() === 'me') {
```
[the remark this answers is not in the document]
Comment (vipulrawat) [reply packages/api/src/routes/profileRoutes.js]: Left intentionally. Didn't make much change on the flow.

[the remark this answers is not in the document]
Comment (bharaninb) [reply packages/api/src/routes/profileRoutes.js]: Then no prob.

**Pull request merged**